static/js/components/LearnerSearch.js

```diff
@@ -99,6 +99,91 @@ export default class LearnerSearch extends SearchkitComponent {
 
   searchkitTranslations: Object = makeSearchkitTranslations();
 
+  renderSearchHeader: Function = (openEmailComposer: Function): React$Element<*>|null => {
+    if (this.getResults() === null) {
+      return null;
+    }
+
+    return (
+      <Grid className="search-header">
+        <Cell col={6} className="result-info">
+          <button
+            id="email-selected"
+            className="mm-button minor-action"
+            onClick={() => openEmailComposer(this.searchkit)}
+          >
+            Email These Learners
+          </button>
+          <HitsStats component={HitsCount} />
+        </Cell>
+        <Cell col={2} />
+        <Cell col={4} className="pagination-sort">
+          <SortingSelector options={sortOptions} listComponent={CustomSortingSelect} />
+          <Pagination showText={false} listComponent={CustomPaginationDisplay} />
+        </Cell>
+        <Cell col={12} className="mm-filters">
+          <SelectedFilters />
+          <ResetFilters component={CustomResetFiltersDisplay}/>
+        </Cell>
+      </Grid>
+    );
+  }
+
+  renderFacets: Function = (): React$Element<*> => {
+    if (this.getResults() === null) {
```

Comment:
there was unexpected issue with here, because  of ```if (!this.getResults())``` facet panel was rendering ambiguously. showing error message.